CheckPart.py

```diff
@@ -0,0 +1,12 @@
+def isFatherAlive(father):
+    if father == "y":
+        return  True
+
+def isMotherAlive(mother):
+        if mother == "y":
+            return True
+
```

Comment:
What's the difference between these two functions they are doing the same thing, even if we change the function name and arg name, they are doing the same. Check that argument == "y".

Moreover, the function name should also be in `snake_case`. Refer back to: https://peps.python.org/pep-0008/